SystemServices/System.json

```diff
@@ -2023,6 +2034,20 @@
                 ]
             }
         },
+	"onRecoveryStateChange":{
```

Comment:
We cannot add newline to the json. The md file generator will break. Pls check and get this fixed